packages/vkui/src/components/CustomSelect/CustomSelect.tsx

```diff
@@ -625,9 +651,54 @@ export function CustomSelect(props: SelectProps) {
     }
   }, [emptyText, options, renderDropdown, renderOption]);
 
+  const clearButton = React.useMemo(() => {
+    if (allowClear && clearButtonProp !== undefined) {
+      return clearButtonProp;
+    }
+
+    if (allowClear && selected?.label) {
```

Comment:
```suggestion
    if (allowClear && nativeSelectValue === '') {
```
так как-будто бы чище

в 248 строчке (....сейчас я понял насколько много кода в селект 😯) такая же проверка

Reply:
Мы здесь проверяем, что какое-то значение выбрано 🤔

Reply:
Ой, сорри...

```diff
- allowClear && nativeSelectValue === ''
+ allowClear && nativeSelectValue !== ''
```

вот так имел ввиду

Reply:
Ааа, там прикол в том, что при неконтролируемом значении `nativeSelectValue` - `undefined`, поправлю